pkg/monitoring/metrics.go

```diff
@@ -40,6 +46,36 @@ func SetReadyGauge(isReady bool) {
 	}
 }
 
+func setKubemacpoolDeployedGauge(isDeployed bool) {
+	if isDeployed {
+		kmpDeployedGauge.Set(1)
+	} else {
+		kmpDeployedGauge.Set(0)
+	}
+}
+
+func TrackMonitoringComponents(conf *cnao.NetworkAddonsConfigSpec, monitoringAvailable bool) error {
+	if !monitoringAvailable {
+		return nil
+	}
+
+	err := trackKubemacpoolMonitoring(conf)
+	if err != nil {
+		return err
+	}
+
+	return nil
```

Comment:
return trackKubemacpoolMonitoring(conf)

Reply:
This is intentional, I want this function to be a placeholder for future alerts.. similar to [Validate()](https://github.com/kubevirt/cluster-network-addons-operator/blob/main/pkg/network/network.go#L27)

perhaps I should do the `errs = append(...)` as well, what do you think?

Reply:
ye maybe then worth to do the append, else its pretty creating infra for air-conditioner :)

Reply:
Done